logstash-core-event-java/build.gradle

```diff
@@ -0,0 +1,104 @@
+buildscript {
+  repositories {
+    mavenLocal()
+    mavenCentral()
+    jcenter()
+  }
+  dependencies {
+    classpath 'net.saliman:gradle-cobertura-plugin:2.2.8'
+    classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.2'
+  }
+}
+
+//allprojects {
```

Comment:
the idea here was that we have multiple subprojects for logstash as we move more things to java. 

If we do not think this will be the case in the near future, lets remove these comments